runtime/kusama/constants/src/lib.rs

```diff
@@ -27,7 +27,8 @@ pub mod currency {
 
 	pub const UNITS: Balance = 1_000_000_000_000;
 	pub const CENTS: Balance = UNITS / 30_000;
-	pub const GRAND: Balance = CENTS * 100_000;
+	pub const QUID: Balance = CENTS * 100;
```

Comment:
I would have assumed that a quid is equivalent to a unit, how is this not the case?

Reply:
Why do you have that assumption?

I don't think we would have two ways to write an amount if they would be the same.

Unit represents "1 KSM"

Whereas "QUID" or "DOLLAR" is some abstract, **non-fiat** representation of what we use to base fees and other storage deposits off of.

When we adjust fees, for example make them cheaper, we simply adjust what `cents` are, and all fees should be adjusted accordingly.

Reply:
Okay the justification that you just told me was non-trivial, sounds like something that we should mention as a comment.

Reply:
Happy to resolve that as a follow up issue. lets not bring that into this big pr

Reply:
https://github.com/paritytech/polkadot/issues/5912